src/Sulu/Component/Content/Types/Rlp/Strategy/RlpStrategy.php

```diff
@@ -270,12 +270,7 @@ public function loadByResourceLocator($resourceLocator, $webspaceKey, $languageC
      */
     public function isValid($path, $webspaceKey, $languageCode, $segmentKey = null)
     {
-        return $path !== '/' && $this->cleaner->validate($path) && $this->mapper->unique(
```

Comment:
Exactly that's the problem. If this function is not used somewhere else, it would not check if the URL is unique. Wondering how the tests are succeeding then 😕